doc/redis.md

```diff
@@ -4,7 +4,7 @@
 To seed a Kubernetes/Openshift cluster with an example Redis resource:
 ```
 $ make cluster/prepare 
-$ make cluster/seed/redis
+$ make cluster/seed/redis PROVIDER=openshift
```

Comment:
same as previous comment - Could we add values for PROVIDER in case AWS and GCP?